drivers/storage/isilon/executor/isilon_executor.go

```diff
@@ -43,6 +44,18 @@ func (d *driver) Name() string {
 	return isilon.Name
 }
 
+// Supported returns a flag indicating whether or not the platform
+// implementing the executor is valid for the host on which the executor
+// resides.
+func Supported(ctx types.Context, opts types.Store) (bool, error) {
+	// make sure mount.nfs is present
+	if err := exec.Command(
```

Comment:
It is definitely assuming that. My (flawed?) reasoning for why this is okay is that libstorage is already making such assumptions for NFS: https://github.com/emccode/libstorage/blob/master/drivers/os/linux/linux.go#L185

Another tactic would be to survey our supported OS's, and see where the binary would be located on each of the, and then just `stat` for the file in all of the locations, returning `true` if any of them were successful.